h/static/scripts/annotator/guest.coffee

```diff
@@ -1,7 +1,20 @@
-Promise = require('es6-promise').Promise
+raf = require('raf')
+Promise = global.Promise ? require('es6-promise').Promise
```

Comment:
Not sure I understand this. Why does using the polyfill have a negative performance impact, when [it's a no-op in environments that already define `Promise`](https://github.com/jakearchibald/es6-promise/blob/1e88f88bab1626911432a507613d63107f252e63/lib/es6-promise/polyfill.js#L21)?